optuna/integration/pytorch_lightning.py

```diff
@@ -42,14 +42,24 @@ def __init__(self, trial, monitor):
         self._trial = trial
         self._monitor = monitor
 
-    def on_epoch_end(self, trainer, pl_module):
+    def _process(self, trainer, pl_module):
         # type: (Trainer, LightningModule) -> None
         logs = trainer.callback_metrics
         epoch = pl_module.current_epoch
-        current_score = logs.get(self._monitor)
+        current_score = logs.get(self.monitor)
```

Comment:
I think `self._monitor` is no longer used. Can we remove it?

Reply:
I think so. I did this change without intention so I consulted PyTorch Lightning 0.7.1. I think we can remove `self._monitor`.
https://github.com/PyTorchLightning/pytorch-lightning/blob/0.7.1/pytorch_lightning/callbacks/early_stopping.py#L51

Reply:
Thank you for your investigation. Then, let's remove it to keep the code clean.